app/views/users/new.html.erb

```diff
@@ -1,25 +1,16 @@
 <div class="w-screen h-full">
 
-  <div class="flex flex-col items-center h-1/2 w-1/4 bg-gray-200 rorunded-3xl p-32" style="margin-top: 8%; margin-left: 37%;">
-    <h2>Register as a New User</h2>
-    <%= form_with model: @user do |f| %>
-    <div class="flex flex-col mb-4">
-      <label for="email" class="text-xl">Email</label>
```

Comment:
Weird, I didn't realize I'd had this form spaced out like this, thanks for fixing it.